core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/resourcebuffer/HttpResourceBufferDAO.scala

```diff
@@ -78,9 +78,9 @@ class HttpResourceBufferDAO(config: ResourceBufferConfig, clientServiceAccountCr
   override def getJobResult(jobId: String): Future[Object] = {
     clientServiceAccountCreds.refreshToken()
     val accessToken = OAuth2BearerToken(clientServiceAccountCreds.getAccessToken)
-      Future {
-        getJobsApi(accessToken).retrieveJobResult(jobId)
-      }
+    Future {
+      getJobsApi(accessToken).retrieveJobResult(jobId)
+    }
```

Comment:
Dunno how this got merged in the first place, but it was causing scalafmt to fail in this PR